### PR TITLE
bug fix  with myfirst commit

### PR DESCRIPTION
**Pull request by emmanueltct**
I pushed here all code in my local repository for progress review

### CODE REVIEW

File: dist/Models/blogComment.js [hunk 1]
```diff
@@ -3,9 +3,10 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
```
Comment (Eli250) [inline dist/Models/blogComment.js]: You shouldn't push dist folder, there is no need. Add dist directory to gitignore.

File: README.md [hunk 2]
```diff
@@ -1 +0,0 @@
-MyBrand portfolio backend part
```
Comment (Eli250) [inline README.md:1]: You need to update your readme file. Include your project title, overview of your project, and how we can manually test your project.

**Review by Eli250 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (emmanueltct) [reply README.md:1]: I worked on it and update my read me file

File: dist/Models/blogComment.js (hunk 1 above)
Comment (emmanueltct) [reply dist/Models/blogComment.js]: I added it to gitignore

**Pull request merged**